pkg/kube/inject/webhook_test.go

```diff
@@ -1459,41 +1381,6 @@ func TestRunAndServe(t *testing.T) {
 	testSideCarInjectorMetrics(t, wh)
 }
 
-func TestReloadCert(t *testing.T) {
```

Comment:
should we keep this test?

Reply:
I will see if I can move it to server tests